src/app/util/attribute-storage.cpp

```diff
@@ -291,6 +292,31 @@ CHIP_ERROR emberAfSetDynamicEndpoint(uint16_t index, EndpointId id, const EmberA
         }
     }
 
+    for (uint8_t i = 0; ep && (i < ep->clusterCount); i++)
+    {
+        if (!ep->cluster)
```

Comment:
Again, if clusterCount > 0 but cluster is null, we crashed back on emberAfClusterCountForEndpointType.

Reply:
Ack.